scripts/jsondeps.gradle

```diff
@@ -21,31 +37,79 @@
 //     Project Name -- first-party (sub)projects
 //   | Package Name Version [Dependency]
 
+
+// TODO: Only print debug logging when running in debug mode?
```

Comment:
I suspect the excessive debug logging of each module artifact for each dependency of each configuration of each subproject will tank performance for particularly-large projects.

Notably, I've seen the output of a customer's project that was over 100 MB(!), and I think this extensive debug logging would substantially slow it down

Reply:
I'm going to try this for now to unblock the fix, and come back to figure out how to selectively debug log later. If we need to back this out in a hurry, we can always comment out the log lines.

Reply:
I've logged the issue in https://github.com/fossas/team-analysis/issues/689.